ff_assignment_portal/ff_assignment_portal/doctype/ff_assignment_submission/ff_assignment_submission.py

```diff
@@ -675,3 +696,19 @@ def compare_hashes(other: dict, original: dict) -> float:
 			score += 1 / len(original)
 
 	return score * 100
+
+def compare_docs(other: dict, original: dict) -> float:
+	"Compares file vectors and returns similarity score percent"
+	from sklearn.feature_extraction.text import TfidfVectorizer
+	from sklearn.metrics.pairwise import cosine_similarity
+	vectorizer = TfidfVectorizer()
+	score = 0
+	for filename, content in original.items():
+		other_content = other(filename,"")
```

Comment:
```suggestion
		other_content = other.get(filename,"")
```

This is how you access a key from a dict. But since it is not a dict, it doesn't work even if this is fixed.